build/images/Dockerfile.scale

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:20.04
```

Comment:
Prefer to use arg type for ubuntu version and default to current ubuntu version when ubuntu version is not provided during docker build e.g docker build --build-arg BASE_IMAGE=ubuntu:21.04 -t my_image . 
 ARG BASE_IMAGE=ubuntu:20.04